test/Tool/signbits6.opt

```diff
@@ -0,0 +1,10 @@
+; REQUIRES: solver
+
+; RUN: %souper-check %solver -print-counterexample=false  %s > %t 2>&1
+; RUN: FileCheck %s < %t
+
+; CHECK: LGTM
+%0:i32 = var (p) (s=31)
+%1:i32 = cttz %0
+%2:i1 = eq %1, 0:i32
+cand %2 1:i1
```

Comment:
I don't understand this one. If there are 31 sign bits, then the lowest bit should be able to be either 0 or 1, right?